planet/src/main/webapp/js/itinerary.js

```diff
@@ -60,8 +80,14 @@ function createItinerary(items) {
     itineraryContainer.innerHTML = '';
 
     items.forEach((item) => {
-        itineraryContainer.innerHTML += '<li>' + item.name + ', ' + item.address + ', ' + timeToString(item.when.start) +
-            ' - ' + timeToString(item.when.end) + '</li>';
+        // Only show one time stamp for hotel.
```

Comment:
this comment doesn't match the if statement. This doesn't need to only apply to hotels, it applies to any zero-length events.

Reply:
fixed!